lib/krikri/harvesters/couchdb_harvester.rb

```diff
@@ -20,6 +20,7 @@ def initialize(opts = {})
       super
       @opts = opts.fetch(:couchdb, view: '_all_docs')
       @opts[:view] ||= '_all_docs'
+      @opts[:limit] ||= 10
```

Comment:
Is this option clear as `limit`?  Would `batch_size` be better?

Limit makes sense down below in the guts of the CouchDB view request, but someone looking at the external interface to the class might comprehend `batch_size` more readily.